apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -67,6 +70,28 @@ const readOrganizations = async ({ context, meters }) => {
     })
 }
 
+const readMeterReportingPeriods = async ({ context, organizations }) => {
+    const organizationIds = organizations.map(org => org.id)
+    return await loadListByChunks({
+        context,
+        list: MeterReportingPeriod,
+        where: {
+            organization: {
+                id_in: organizationIds,
+            },
+        },
+    })
+}
+
+const checkIsDateInPeriod = (date, today, start, end) => {
+    return dayjs(date).isBetween(
+        today.format('YYYY-MM-DD').slice(0, -2) + (start.toString().length === 1 ? '0' + start : start),
```

Comment:
You can do that trick without `slice` call
```suggestion
        today.format('YYYY-MM-') + (start.toString().length === 1 ? '0' + start : start),
```